### PR TITLE
Ping pong

### PR DESCRIPTION
**Pull request by idanmel**
Wrote an endpoint /ping that returns pong.
I also handle 404, to show that only /ping returns pong

### CODE REVIEW

**Comment by idanmel** [comment]
Wrote an endpoint /ping that returns pong.
I also handle 404, to show that only /ping returns pong

File: clojure-hacker-news.iml [hunk 1]
```diff
@@ -17,10 +17,29 @@
     </content>
```
Comment (ytmlndr) [inline clojure-hacker-news.iml:17]: we add `*.iml` files to the `.gitignore` as these files serve IntelliJ specific configuration

Comment (SmartassSkeleton) [reply clojure-hacker-news.iml:17]: You can generate pretty wholesome `.gitignore`s using gitignore.io. Just add your editor/s and the languages used. You'll never have to think of those anymore ;)

Comment (idanmel) [reply clojure-hacker-news.iml:17]: Thanks, removed the iml file in a separate PR.

**Pull request merged**